aten/src/ATen/FunctionalStorageImpl.h

```diff
@@ -147,6 +169,10 @@ struct TORCH_API FunctionalStorageImpl : public c10::StorageImpl {
   // storage cannot be unfrozen.
   bool frozen_ = false;
 
+  bool inductor_storage_resized_ = false;
+  int64_t original_storage_size_;
+  int64_t curr_storage_size_;
```

Comment:
needs docs

Reply:
Make these SymInts, it's easy and saves us some dynamic shapes enablement later.